TheWarWithin/Priorities/DruidBalance.simc

```diff
@@ -5,7 +5,7 @@ actions.precombat+=/variable,name=on_use_trinket,value=0
 actions.precombat+=/variable,name=on_use_trinket,op=add,value=(trinket.1.has_proc&trinket.1.cooldown.duration|trinket.1.is.spymasters_web|trinket.1.is.signet_of_the_priory)&!trinket.1.is.arakara_sacbrood
 actions.precombat+=/variable,name=on_use_trinket,op=add,value=((trinket.2.has_proc&trinket.2.cooldown.duration|trinket.2.is.spymasters_web|trinket.2.is.signet_of_the_priory)*2&!trinket.2.is.arakara_sacbrood)
 ## Regrowth to stack for Keeper of the Grove...
-actions.precombat+=/regrowth,if=hero_tree.keeper_of_the_grove&!talent.stellar_flare&active_dot.regrowth=0
+actions.precombat+=/regrowth,if=hero_tree.keeper_of_the_grove&!talent.stellar_flare&active_dot.regrowth=0&!prev.1.regrowth&!prev.1.wrath
```

Comment:
I *think* this would work as-is if Regrowth were flagged to check as a "dot" -- 

```Lua
    regrowth = {
        id = 8936,
        duration = function () return mod_circle_hot( 12 ) end,
        tick_time =  function () return mod_circle_hot( 2 ) end,
        type = "Magic",
        max_stack = 1,
+        dot = "buff",
+        friendly = true
    },
```

Reply:
Okay, will test!

Reply:
Yes, it does count it as an active dot if you do that